projects/aca-content/src/lib/components/libraries/libraries.component.ts

```diff
@@ -54,6 +59,15 @@ export class LibrariesComponent extends PageComponent implements OnInit {
   ngOnInit() {
     super.ngOnInit();
 
+    this.extensions
```

Comment:
can be moved to base class, also, why removing certain actions in code? should be removed from the extensions json file?